samples/scaleout/senderside/sample.md

```diff
@@ -49,16 +50,14 @@ Contains message definitions shared by the server and the clients.
 
 ### Unaware client
 
-This project contains a code for a client which is not aware of specific scale out design of the server. The client specifies only the logical routing
+This project contains code for a client which is not aware of specific scale out design of the server. The client specifies only the logical routing:
 
 snippet:Logical-Routing
 
-NOTE: When using MSMQ it means NServiceBus treats MSMQ as a *local broker* and all queues are assumed to be on the local machine
-
 
 ### Aware client
 
-In addition to the logical routing, the scale out-aware client uses a routing file to tell NServiceBus about the specific instances of the Server
+In addition to the logical routing, the scale out-aware client uses a [instance mapping file](/nservicebus/msmq/routing.md) to tell NServiceBus about the specific instances of the Server:
```

Comment:
Should `Server` be capitalized?

Also, how about:

In addition to the logical routing, the scale out-aware client uses a [instance mapping file](/nservicebus/msmq/routing.md) to tell NServiceBus about the specific instances of the server. This is done as shown below: